app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;} 
+a:visited {color:#000;}  
+a:hover   {color:#000;}  
+a:active  {color:#000;}
+
+body {
+  background-color: #F0F0F0;
+  font-family: sans-serif;   
+}
+
+.login {
+  position: relative;
+  max-width: 250px;
+  background-color: white;
+  margin: 10px auto;
+  margin-top: 10%;
+  margin-bottom: 10px;
+  padding: 10px 10px;
+  border-style: solid;
+  border-color: #E0E0E0;
+  border-width: 1px;
+}
+
+.login-heading {
+  position: relative;
+  max-width: 14%;
+  margin: 0 auto;
+  margin-left: 5px;
+}
+
+.heading {
+  position: relative;
+  max-width: 14%;
+  margin: 30px auto;
+}
+
+.timeline {
+  position: relative;
+  max-width: 604px;
+  padding: 20px auto;
+  margin: 10px auto;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+.post {
+  background-color: white;
+  margin: 10px auto;
```

Comment:
Properties should be ordered background-color, border-color, border-style, border-width, margin, margin-bottom, margin-top